02-make-macos.sh

```diff
@@ -2,18 +2,20 @@
 set -e
 PAT_ROOT="../pat"
 VERSION=$(grep "Version =" ${PAT_ROOT}/internal/buildinfo/VERSION.go|cut -d '"' -f2)
+ARCH=$(arch)
```

Comment:
Would it be better to use `go env GOARCH` instead?

Running `arch` on my iMac running Monterey reports "i386" 🤔

Reply:
Good call, done.